lib/internal/main/test_runner.js

```diff
@@ -5,18 +5,23 @@ const {
 } = primordials;
 
 const {
-  prepareMainThreadExecution,
   markBootstrapComplete,
+  prepareTestRunnerMainExecution,
 } = require('internal/process/pre_execution');
 const { isUsingInspector } = require('internal/util/inspector');
 const { run } = require('internal/test_runner/runner');
 const { parseCommandLine } = require('internal/test_runner/utils');
+const { getOptionValue } = require('internal/options');
 const { exitCodes: { kGenericUserError } } = internalBinding('errors');
 let debug = require('internal/util/debuglog').debuglog('test_runner', (fn) => {
   debug = fn;
 });
 
-prepareMainThreadExecution(false);
+const isTestIsolationDisabled = getOptionValue('--test-isolation') === 'none';
```

Comment:
Can we/should we move `const options = parseCommandLine();` to above `prepareTestRunnerMainExecution` and use `isolation` from there, to avoid using `getOptionValue` here?

@cjihrig cc

Reply:
ParseCommandLine, if I'm not mistaken, can't be used before init! That's why I had to make a redundant call.

I'll take another look just to be sure and report back here with the result

Reply:
Ah, this is in `lib/internal/main/test_runner.js`... that should be fine then since that file has nothing to do with the `run()` API.

Reply:
My bad 😬 we can refactor it to avoid using `getOptionValue`.  
I'm running the tests, once I have a local green flag, I'll push a fixup commit!  
Thanks @atlowChemi for catching it!